### DISCUSSION TITLE
Zoom in on headings on mobile

### DISCUSSION THREAD

**Discussion by fernandopirescoelho** [vslinko/obsidian-zoom#80] [General]
Hi,

It  is possible now to zoom in into bullets on iOS by clicking the relevant bullet.
How can we zoom into headings?  In desktop you have the CTRL + . shortcut. But I did not figure out how can we zoom in on headings that doesn’t (cannot) have a preceding bullet.

**Comment by Fyzu** [comment]
You can use command from toolbar panel

**Comment by fernandopirescoelho** [comment]
Brilliant!!!


Kind Regards,
Fernando Coelho
________________________________
